### DISCUSSION TITLE
Omnik datalogger Persistant_cache.json layout

### DISCUSSION THREAD

**Discussion by DS723** [jbouwh/omnikdatalogger#95] [Q&A]
All, 
Trying to set ip the TCP Omnik datalogger   and there is a reference made to the persistant_cache.json file. Does any one know the lay out of this file ?    Thnx for the help

**Comment by jbouwh** [comment]
Hi, see this message here. The file should be created automatically.

**Comment by DS723** [comment]
Dank voor het antwoord   

 

Met vriendelijke groet,

 

Jos Brakenhoff

 

Van: Jan Bouwhuis ***@***.***> 
Verzonden: vrijdag 10 januari 2025 22:33
Aan: jbouwh/omnikdatalogger ***@***.***>
CC: DS723 ***@***.***>; Author ***@***.***>
Onderwerp: Re: [jbouwh/omnikdatalogger] Omnik datalogger Persistant_cache.json layout (Discussion #95)

 

Hi, see this message here. The file should be created automatically.
